components/constant_contact/actions/update-contact/update-contact.mjs

```diff
@@ -137,12 +219,12 @@ export default {
       ...data
     } = this;
 
-    const contact = await constantContact.getContact({
+    const contact = await this.constantContact.getContact({
```

Comment:
Since `constantContact` is declared as a constant, I think `this` can be removed.
```
const contact = await constantContact.getContact({
   $,
   contactId,
});